kong/tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import os
+
+from datadog_checks.dev import docker_run
+
+from . import common
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    """
+    Start a kong cluster
+    """
+    compose_directory = os.path.join(common.HERE, 'compose')
+
+    with docker_run(
+        compose_file=os.path.join(compose_directory, 'docker-compose.yml'),
+        env_vars={'COMPOSE_DIRECTORY_PATH': compose_directory}
```

Comment:
Could we use `endpoints=common.STATUS_URL` here and remove the `wait_for_cluster` method?